data/coalition/heliarch intro.txt

```diff
@@ -1537,7 +1537,7 @@ mission "Heliarch Containment 4-B"
 		payment 495000
 		conversation
 			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortub. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`
-			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Very useful to us your support has been, Captain."`
+			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Been very useful to us your support has, Captain."`
```

Comment:
```suggestion
			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent to help contain it. What remains of the fleet is helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Been very useful to us your support has, Captain."`
```

or

```suggestion
			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent to help contain it. The remaining ships are helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Been very useful to us your support has, Captain."`
```